cl_sii/dte/constants.py

```diff
@@ -6,6 +6,187 @@
 
 """
 import enum
+from datetime import date
+from typing import FrozenSet
+
+
+@enum.unique
+class TipoDocumentoEnum(enum.IntEnum):
```

Comment:
- Evaluate merging with `cl_sii.dte.constants.TipoDocumentoEnum` and/or `cl_sii.rcv.constants.RcvTipoDocto`. That would be a big change, so I recommend doing it in a separate pull request.
- Remove `Enum` suffix from class name.
- Consider moving it to `cl_sii.base.constants` because it looks like it could be useful across multiple SII subpackages.